pkg/agent/datamodel/defaults.go

```diff
@@ -517,8 +517,6 @@ func (cs *ContainerService) setOrchestratorDefaults(isUpgrade, isScale bool) {
 
 		// Configure controller-manager
 		cs.setControllerManagerConfig()
```

Comment:
this is also not needed

Reply:
It is removed in this PR: https://github.com/Azure/AgentBaker/pull/194